Makefile.kyverno.mk

```
@@ -0,0 +1,10 @@
+KYVERNO_CLI_IMAGE       ?= ghcr.io/kyverno/kyverno-cli:latest
+KYVERNO_CLI_DOCKER      ?= $(DOCKER_CMD) $(DOCKER_ARGS) $(root_volume) --entrypoint=/kyverno $(KYVERNO_CLI_IMAGE)
+KYVERNO_CLI_VEROBOSITY  ?= -v2
```

I'm guessing there can be more args to append? Why not make it more generic?
```suggestion
KYVERNO_CLI_ARGS  ?= -v2
```